### PR TITLE
- Added Multiselect to form

### PR DESCRIPTION
**Pull request by KaiHotz**
- Frontend finished first edition for Project creation form. Not yet connected to backend
captura de pantalla 2017-10-22 a la s 19 58 48

### CODE REVIEW

**Comment by lorenzosinisi** [comment]
Please dont merge now

File: assets/js/containers/ProjectNew.js [hunk 1]
```diff
@@ -45,7 +71,24 @@ class ProjectNew extends Component {
       {
         address: ''
       }
-    ]
+    ],
+    reposInit: [
+      {
+        owner: 'KaiHotz',
```
Comment (lorenzosinisi) [inline assets/js/containers/ProjectNew.js]: This needs to come from the APIs tho, no?

**Comment by KaiHotz** [comment]
don't worry , i merge only as soon as you approve

File: assets/js/containers/ProjectNew.js [hunk 2]
```diff
@@ -102,16 +172,64 @@ class ProjectNew extends Component {
               onChange={this.handleChange('name')}
               />
           </Grid>
+
           <Grid item xs={12} sm={6}>
-            <Typography type='display1'>
-              Multi Select goes Here :
+            <Typography type='headline' className={selectorsTitle}>
+              Chose the Repos for your Project :
```
Comment (lorenzosinisi) [inline assets/js/containers/ProjectNew.js]: Please use "Control repos from this Project"

File: assets/js/utils/index.js [hunk 3]
```diff
@@ -23,3 +23,18 @@ export const errorMessage = (error) => {
...
+
+export const getSelectValuesIndex = (select) => {
+  let result = []
+  let options = select && select.options
+  let opt
+
+  for (let i = 0, iLen = options.length; i < iLen; i++) {
+    opt = options[i]
+
+    if (opt.selected) {
```
Comment (lorenzosinisi) [inline assets/js/utils/index.js]: Is there any way to avoid this if?

File: assets/package.json [hunk 4]
```diff
@@ -20,6 +20,7 @@
     "axios": "^0.16.2",
     "history": "^4.7.2",
     "immutable": "^3.8.2",
+    "lodash": "^4.17.4",
```
Comment (lorenzosinisi) [inline assets/package.json]: 💯

File: assets/js/utils/index.js (hunk 3 above)
Comment (KaiHotz) [reply assets/js/utils/index.js]: not that i know of

Comment (lorenzosinisi) [reply assets/js/utils/index.js]: `
  opt.selected && result.push(opt.index)
`
For example?

File: assets/js/containers/ProjectNew.js (hunk 1 above)
Comment (KaiHotz) [reply assets/js/containers/ProjectNew.js]: yes, this is just mocked data, will be replaced through a selector in props once API is ready and repos are on the store

Comment (KaiHotz) [reply assets/js/containers/ProjectNew.js]: we could create a separated task for that, like "Connect Form with Backend"

File: assets/js/utils/index.js (hunk 3 above)
Comment (KaiHotz) [reply assets/js/utils/index.js]: no, this wont work because && is a Logical comparison,

Comment (KaiHotz) [reply assets/js/utils/index.js]: No wait my Bad , yes this should work

**Comment by lorenzosinisi** [comment]
This should not be merged anymore once I have merged this into https://github.com/lorenzosinisi/scrumpointer/pull/33. Have a look there please @KaiHotz

**Pull request merged**